plugins/modules/usb_facts.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2024, Max Maxopoly <max@dermax.org>
+#
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: usb_facts
+short_description: Allows listing information about usb devices
+description:
+  - Allows retrieving information about available usb devices through lsusb
```

Comment:
```suggestion
short_description: Allows listing information about USB devices
description:
  - Allows retrieving information about available USB devices through C(lsusb).
```
Similar further below in return value descriptions.